index.html

```diff
@@ -10,6 +10,21 @@
   </head>
 
   <body>
-    <h1 class="title">Hello World</h1>
+    <div>
```

Comment:
I would suggest you to wrap all this component on a <div> that could be used as a container to make it more reusable when adding more books.